### PR TITLE
Updating Zew Zealand electorates.

### PR DESCRIPTION
**Pull request by novitam**
Updating the general electorates for New Zealand.

Authoritative source: https://vote.nz/boundary-review-2019-2020/what-are-the-main-changes/
Summary of changes: https://vote.nz/assets/Boundary-Review/SCHEDULE-B-LIST-OF-UNCHANGED-AND-CHANGED-ELECTORATES.pdf

### CODE REVIEW

**Comment by novitam** [comment]
Please review @jloutsenhizer

File: identifiers/country-nz/parliament_electorates.csv [hunk 1]
```diff
@@ -1,73 +1,82 @@
...
-ocd-division/country:nz/maori_electorate:ikaroa-rawhiti,Ikaroa-Rawhiti
-ocd-division/country:nz/maori_electorate:tamaki_makaurau,Tamaki Makaurau
-ocd-division/country:nz/maori_electorate:te_tai_hauauru,Te Tai Hauauru
-ocd-division/country:nz/maori_electorate:te_tai_tokerau,Te Tai Tokerau
-ocd-division/country:nz/maori_electorate:te_tai_tonga,Te Tai Tonga
-ocd-division/country:nz/maori_electorate:waiariki,Waiariki
+id,name,validFrom,validTo
+ocd-division/country:nz,New Zealand,,
+ocd-division/country:nz/general_electorate:auckland_central,Auckland Central,2017-11-07,
+ocd-division/country:nz/general_electorate:bay_of_plenty,Bay of Plenty,2017-11-07,2020-09-19
```
Comment (jloutsenhizer) [inline identifiers/country-nz/parliament_electorates.csv:4]: I believe Bay of plenty is still a valid electoral district, its borders have only changed.

File: identifiers/country-nz/parliament_electorates.csv [hunk 2]
```diff
@@ -1,73 +1,82 @@
...
+ocd-division/country:nz/general_electorate:botany,Botany,2017-11-07,
+ocd-division/country:nz/general_electorate:christchurch_central,Christchurch Central,2017-11-07,
+ocd-division/country:nz/general_electorate:christchurch_east,Christchurch East,2017-11-07,
+ocd-division/country:nz/general_electorate:clutha-southland,Clutha-Southland,2017-11-07,2020-09-19
+ocd-division/country:nz/general_electorate:southland,Southland,2020-09-19,
+ocd-division/country:nz/general_electorate:coromandel,Coromandel,2017-11-07,
+ocd-division/country:nz/general_electorate:dunedin_north,Dunedin North,2017-11-07,2020-09-19
```
Comment (jloutsenhizer) [inline identifiers/country-nz/parliament_electorates.csv:11]: I believe these and many others of the electoral districts were valid before 2017-11-07:
https://en.wikipedia.org/wiki/Dunedin_North_(New_Zealand_electorate)

validFrom should reflect either when the electoral districts were created or they can be left blank.

File: identifiers/country-nz/parliament_electorates.csv (hunk 1 above)
Comment (novitam) [reply identifiers/country-nz/parliament_electorates.csv]: Good catch. Added.

File: identifiers/country-nz/parliament_electorates.csv (hunk 2 above)
Comment (novitam) [reply identifiers/country-nz/parliament_electorates.csv]: Removed validFrom dates of existing electoral districts.

**Comment by novitam** [comment]
Done. PTAL.

**Comment by novitam** [comment]
Good call. Missed that one since its a completely new electorate. Added.

**Comment by jpmckinney** [comment]
Usually validTo/From are set to different dates (a day apart) because the dates are inclusive of the range, not exclusive. Presently, from the perspective of 2020-09-19, for example, you'd have too many valid divisions, as both the divisions starting and ending on that date would be included.

**Comment by novitam** [comment]
Since the general election is held on 19 Sept 2020, I change the ValidFrom dates for the new electorates to be 20 Sept 2020. Is that OK?

**Comment by novitam** [comment]
Done.

**Comment by jloutsenhizer** [comment]
Latest changes still look good to me. Merging.

**Pull request merged**